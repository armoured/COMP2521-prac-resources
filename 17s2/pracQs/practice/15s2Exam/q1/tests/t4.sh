./q1 1 < tests/tree2
