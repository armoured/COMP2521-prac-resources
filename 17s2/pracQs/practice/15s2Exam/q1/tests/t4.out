Original Tree:
  2
 / \
1   3
     \
      4
       \
        5

After Mapping:
  1
 / \
0   2
     \
      3
       \
        4
